src/__stories__/Utilities/Badge.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import { withKnobs } from '@storybook/addon-knobs';
+import { select, text } from '@storybook/addon-knobs/react';
+import { storiesOf } from '@storybook/react';
+
+import omitBy = require('lodash/omitBy');
+import { Badge, BadgeColor, BadgeVariant } from '../../Badge';
+
+export const badgeKnobs = (tabName = 'Badge'): any => {
+  const props = {
+    label: text('label', '5', tabName),
+    color: select('color', [BadgeColor.Error, BadgeColor.Warning, BadgeColor.Default], BadgeColor.Default, tabName),
+    variant: select('variant', [BadgeVariant.Pill, BadgeVariant.Textual], BadgeVariant.Pill, tabName),
+  };
+
+  return omitBy(props, val => !val);
```

Comment:
`omitBy` is not needed here. It's some leftover, new code shouldn't really use it because it has no affect in most of the cases (including yours).

Reply:
👍